.github/workflows/integration-tests.yaml

```diff
@@ -31,4 +32,4 @@ jobs:
 
       - name: Run integration tests
         run: |
-          python -m unittest discover -s process_report/tests/integration
+          python -m pytest -q process_report/tests/integration
```

Comment:
I would remove the `-q` flag here, since I think it's better to keep normal verbosity.